jsapp/js/components/modalForms/formMedia.es6

```diff
@@ -0,0 +1,277 @@
+import React from 'react';
+import autoBind from 'react-autobind';
+import alertify from 'alertifyjs';
+import Dropzone from 'react-dropzone';
+import TextBox from 'js/components/common/textBox';
+import {actions} from '../../actions';
+import {bem} from 'js/bem';
+
+/*
+ * Modal for uploading form media
+ */
+class FormMedia extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      uploadedAssets: null,
+      fieldsErrors: {},
+      inputURL: '',
+      // to show loading icon instead of nothing on first load
+      isVirgin: true,
```

Comment:
Let's change this to something neutral in meaning, maybe `isReady` (mind you that then the boolean values would be opposite). I talked with John recently as I felt this naming could be hurtful and decided we shouldn't use it :heart: I will change the few other places that use it :v: